lib/k8s/middleware/request/initialize.ex

```diff
@@ -9,7 +9,7 @@ defmodule K8s.Middleware.Request.Initialize do
   def call(%Request{conn: conn, method: method, headers: headers, opts: opts} = req) do
     with {:ok, request_options} <- K8s.Conn.RequestOptions.generate(conn) do
       request_option_headers = conn.http_provider.headers(method, request_options)
-      updated_headers = Keyword.merge(headers, request_option_headers)
+      updated_headers = headers ++ request_option_headers
```

Comment:
Ahm... I didn?t get which one you prefer... keyword lists or lists of binary/binary tuples?

Reply:
I do agree. Looking at the [typespec of HTTPoison headers](https://hexdocs.pm/httpoison/HTTPoison.Request.html#t:headers/0), keyword lists should be fine, no?
I was confused that the typespec (of %Request{}) said keyword list while the code did b/b tuples. I think it would even have raised an exception if request.headers was not empty (Keyword.merge/2 does not work for b/b tuples). But I'll change this to keyword lists.